css/60_photos.css

```diff
@@ -397,6 +399,13 @@ label.panoramax-hd {
 }
 
 
+#ideditor-viewer-mapilio-simple img {
```

Comment:
this block will duplicate the styles added in #9849, so it should probably be removed